app/src/main/java/com/framgia/fsalon/utils/binding/BindingUtils.java

```diff
@@ -22,13 +25,15 @@
 import com.framgia.fsalon.data.model.Stylist;
 import com.framgia.fsalon.screen.booking.BookingViewModel;
 import com.framgia.fsalon.utils.LayoutManager;
+import com.github.mikephil.charting.animation.Easing;
```

Comment:
ERROR: Unused import: com.github.mikephil.charting.animation.Easing. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports